src/index.html

```diff
@@ -4,10 +4,349 @@
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated
+              in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p>
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than
+              200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this
+              area better than other specialists.
+              Working directly with developers - so have more useful information
+              about the profitable deals.
+            </p>
+            <strong class="info__text info__text-decoration">
+              We will never be late!
+            </strong>
+            <strong class=" info__text info__text-decoration">
+              Guaranteed secure transaction - 100%
+            </strong>
+          </div>
+        </div>
+      </section>
+
+      <section class="section" id="gallery">
+        <h2 class="heading heading__position section__heading">Gallery
+          <span class="heading__logo"></span>
+        </h2>
+        <div class="gallery">
+          <div class="photos photos--right">
+            <img
+              src="./images/photos/1.jpg"
+              alt="" class="photos__img"
+            >
+            <img
+              src="./images/photos/2.jpg"
+              alt="" class="photos__img"
+            >
+          </div>
+          <p class="gallery__address">
+            <a
+              href="https://www.google.com/maps/place/%D0%9D%D0%BE%D1%80%D1%82+%D0%9C%D0%B0%D1%8F%D0%BC%D0%B8+%D0%91%D0%B8%D1%87,+%D0%A4%D0%BB%D0%BE%D1%80%D0%B8%D0%B4%D0%B0+33162,+%D0%A1%D0%A8%D0%90/@25.9289279,-80.1973892,14z/data=!3m1!4b1!4m5!3m4!1s0x88d9addb1b652001:0x525bd42381a2a4cf!8m2!3d25.9260794!4d-80.1819268?hl=ru"
+              class="gallery__link"
+            >
+              Marina Palms
+            </a>
+            / North Miami Beach, FL 33162
+          </p>
+
+          <div class="photos">
+            <img
+              src="./images/photos/4.jpg"
+              alt="" class="photos__img"
+            >
+            <img
+              src="./images/photos/3.jpg"
+              alt="" class="photos__img"
+            >
+          </div>
+          <p class="gallery__address">
+            <a
+              href="https://www.google.com/maps/place/%D0%9C%D0%B0%D0%B9%D0%B0%D0%BC%D0%B8-%D0%91%D0%B8%D1%87,+%D0%A4%D0%BB%D0%BE%D1%80%D0%B8%D0%B4%D0%B0+33139,+%D0%A1%D0%A8%D0%90/@25.7819398,-80.1904187,13z/data=!3m1!4b1!4m5!3m4!1s0x88d9b4661cbb1ae5:0x557bd4f5339da98e!8m2!3d25.783312!4d-80.1484407?hl=ru"
+              class="gallery__link"
+            >
+              Edition Residens
+            </a>
+            / Miami Beach, FL 33139
+          </p>
+
+          <div class="photos photos--left">
+            <img
+              src="./images/photos/5.jpg"
+              alt="" class="photos__img"
```

Comment:
```suggestion
            <img
              src="./images/photos/5.jpg"
              alt=""
              class="photos__img"
```
Так же с атрибутами. Либо все переносим, либо все пишем в одной строке. Кстати не оставляем alt пустым, уже обсуждали это